.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,44 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.245.0/containers/typescript-node
+{
+	"name": "CipherEditor Dev Container Suite",
```

Comment:
The branding guidelines (which aren't public as of know) fixed the spelling of the app's name to be 'ciphereditor' in all lowercase letters.